src/utility/setLocalStorage.tsx

```diff
@@ -0,0 +1,30 @@
+import { Settings } from '../settings/settingsConfig';
+
+class LocalStorage {
+  isGeneratedMeals: boolean;
+
+  constructor(isGeneratedMeals: boolean) {
+    this.isGeneratedMeals = isGeneratedMeals;
+  }
+  saveSettings(storage: Settings) {
```

Comment:
Note: I'd name this parameter data or settings, but not storage

Reply:
renamed